guide/preparations/setting-up-a-bot-application.md

```diff
@@ -7,9 +7,9 @@ Now that you've installed Node, discord.js, and hopefully a linter, you're almos
 It's incredibly easy to create one. The steps you need to take are as follows:
 
 1. Open up [the Discord website](https://discordapp.com/) and [login](https://discordapp.com/login).
-2. Hover over the "Developers" drop-down menu and click on the [Developer Portal](https://discordapp.com/developers/docs/intro) link.
-3. On the header click on the [Applications](https://discordapp.com/developers/applications) link.
-4. Click on the "Create an application" button.
+2. Hover over the "Developers" drop-down menu and click on the [Developer Portal](https://discordapp.com/developers/applications/) link.
+3. Click on the "New Application" button.
+4. Enter a name and select team to create a bot in if you joined any team. Then confirm the pop-up window by clicking the "Create" button.
```

Comment:
I think the wording is a bit off and rather hard to comprehend for step 4, I'd suggest something like the following:

```suggestion
4. Enter a name and optionally select a team (if you want the bot to belong to one). Then confirm the pop-up window by clicking the "Create" button.
```